babel.config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  presets: ["@vue/cli-plugin-babel/preset"]
+  presets: [["@babel/preset-env", { useBuiltIns: "entry", corejs: 3 }]]
```

Comment:
We only need to change the useBuiltIns value. By default, vue cli-plugin-babel/preset sets the useBuiltIns to  `usage`. If we just update it to use `entry`, we are good. 

`  presets: [["@vue/cli-plugin-babel/preset", { useBuiltIns: "entry" }]]`

Reply:
Cool, amended my last commit